scripts/add_fossils.py

```
@@ -287,43 +342,234 @@ def backfill_fossils(data_dir, repo_urls):
+                    {"snapshots": snapshots, "fossils": fossils},
+                    f,
+                    separators=(",", ":"),
+                )
+            os.replace(tmp_file, json_file)
 
-            logger.info(f"  ✓ Successfully wrote fossils for {repo_name}")
+            logger.info("  ✓ Successfully wrote fossils for %s", repo_name)
 
-        except Exception as e:
-            logger.error(f"  ✗ Error computing fossils for {repo_name}: {e}")
+        except Exception as e:  # pylint: disable=broad-exception-caught
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)